### ISSUE TITLE
Fill movie properties from IMDb id?

### ISSUE THREAD

**Issue by benthamite** [isamert/orgmdb.el#2]
I really like this package! Is there a way to fill movie properties using the unique IMDb id? I'm unable to get orgmdb identify the right entry in some cases, typically involving series episodes ([example](https://www.imdb.com/title/tt11062202/)).

**Comment by isamert** [comment]
Thanks for your kind words! 

I've been also having this issue but haven't got around to fix it. Now I added the following function: `orgmdb-fill-movie-properties-with-imdb-id` which asks you for the IMDb id interactively and fills based on that. Also `orgmdb-fill-movie-properties` is more intelligent now, it detects IMDb id from the following formats:

```
* tt01234567

* Title
:PROPERTIES:
:IMDB-ID: tt01234567
:END:
```

The first one does not really makes sense to use it, because after you fill properties the title will still remain as the IMDb id. So I added a variation to this function, if you call `orgmdb-fill-movie-properties` with the prefix arg (that is, pressing `Ctrl-u` before calling this function) it will also format the title.
